lib/src/components/NxDropdown/NxDropdown.scss

```diff
@@ -8,57 +8,80 @@
 @import '../../scss-shared/nx-text-helpers';
 @import '../../scss-shared/nx-container-helpers';
 
+$nx-dropdown-width: 250px;
+
 .nx-dropdown {
   display: inline-block;
   position: relative;
+  width: $nx-dropdown-width;
 }
 
 .nx-dropdown__toggle {
+  display: flex;
   margin: 0;
   outline: none;
+  text-align: left;
+  padding-right: $nx-spacing-md;
+  padding-left: $nx-spacing-md;
+  width: $nx-dropdown-width;
+
+  span {
+    flex-grow: 1;
+  }
+
+  .nx-icon {
+    align-self: center;
+    width: 12px;
+  }
 }
 
 .nx-dropdown-menu {
   @include container-vertical();
 
   background: $nx-off-white;
   border: $nx-border-dark;
-  border-radius: 4px;
-  box-shadow: -4px 4px 10px 0 $nx-slate;
+  border-radius: $nx-border-radius;
+  box-shadow: 0 2px 12px 0 rgba(74, 77, 113, 0.2),
+              0 1px 2px 0 rgba(91, 95, 141, 0.2),
+              0 1px 1px 0 rgba(124, 128, 179, 0.2);
   color: $nx-text-color-light;
   display: block;
-  margin: 6px 0 0 0;
-  max-width: 300px;
-  min-width: 160px;
-  padding: 8px 12px;
+  margin: $nx-spacing-xxs 0 0 0;
+  padding: $nx-spacing-xs 0;
   position: absolute;
   top: 100%;
+  width: $nx-dropdown-width;
 }
 
 .nx-dropdown-link, .nx-dropdown-button {
+  @include container-horizontal;
   @include truncate-ellipsis();
 
   cursor: pointer;
   display: block;
   height: 32px;
   line-height: 32px;
+  padding: 0 $nx-spacing-md;
   text-decoration: none;
 }
 
 .nx-dropdown-button {
   background-color: transparent;
-  border-color: transparent;
+  border: none;
   border-radius: 0;
   box-sizing: border-box;
   color: $nx-text-color-light;
   font-size: 14px;
```

Comment:
Seems like this shouldn't be here

Reply:
> Seems like this shouldn't be here

https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/43cb4f4437f30fcfb8d9dfbd35f8ae554a222c8f

Reply:
> 43cb4f4

That's not what I meant.  The `font-size` shouldn't be here.

Reply:
> > [43cb4f4](https://github.com/sonatype/sonatype-react-shared-components/commit/43cb4f4437f30fcfb8d9dfbd35f8ae554a222c8f)
> 
> That's not what I meant. The `font-size` shouldn't be here.

Ah, https://github.com/sonatype/sonatype-react-shared-components/pull/55/commits/b27f93b17038a124c411ebff792eec71c60c36a0